stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -750,7 +749,7 @@ extension Unsafe${Mutable}BufferPointer {
   @inlinable
   @_alwaysEmitIntoClient
   @available(*, deprecated, renamed: "update(repeating:)")
-  @_silgen_name("_deprecated_assign_repeating")
+  @_silgen_name("_swift_se0370_UnsageMutableBufferPointer_assign_repeating")
```

Comment:
Small typo
```suggestion
  @_silgen_name("_swift_se0370_UnsafeMutableBufferPointer_assign_repeating")
```

Reply:
@glessard Should these deprecated/renamed APIs be `@inline(__always)` instead of `@inlinable`?

Reply:
@benrimmington They're `@_alwaysEmitIntoClient`, which is always.

Reply:
> > Should these deprecated/renamed APIs be `@inline(__always)` instead of `@inlinable`?
>
> They're `@_alwaysEmitIntoClient`, which is does that.

Are you sure? See this comment by @Catfish-Man: https://github.com/apple/swift/pull/60025#discussion_r919550276

I think adding `@inline(__always)` would mean that, after recompiling, the deprecated APIs aren't used at runtime. They would become "aliases" or "macros" for the new APIs.

Reply:
The important part is to not add ABI, thus `@_alwaysEmitIntoClient`; in `-O`, the optimizer will see that it's a 1:1 replacement and will do the right thing. I'm not sure we need to worry about `-Onone`.